src/shared/date/starting-years.js

```diff
@@ -0,0 +1,13 @@
+// equivalents of 1970, except for Nepali which is 3 years later
+// we don't have calendar details for nepali calendar from 1970 and back
+
+export const startingYears = {
```

Comment:
* Is this an exhaustive list?
* Or does it cover all starting years relevant to dhis2?
* Should this be moved to the multi-calendar-dates library or stay here?